### PR TITLE
This needed to be done

### LINKED ISSUE #1: This does nothing
### Expected behavior
It should do something other than print 'wtf' and exit.

### Actual behavior
It prints 'wtf' and exits.

### PR DESCRIPTION
**Pull request by ramdesh**
I know I will be ostracized in the community for doing this, but it needs to be done. I can't stand by idly while despotism and ignorance defiles the good name of the World Taekwondo Federation.

### CODE REVIEW

File: wtfsh.c [hunk 1]
```diff
@@ -8,6 +8,6 @@ int main(void) {
   char *newenv[] = { NULL };
   char *line = readline("$ ");
   execve(line, newargv, newenv);
-  perror("wtf");
+  perror("WTF");
```
Comment (aweeraman) [inline wtfsh.c:11]: Wrong. Please correct. You're going to have to read my mind on this one.

**Comment by aweeraman** [comment]
Note to self: from the same idiot who opened issue #1.

**Comment by ramdesh** [comment]
This isn't over.

**Comment by aweeraman** [comment]
Bring it on loser

**Pull request closed without merging**